Sources/SegmentAmplitude/AmplitudeSession.swift

```diff
@@ -115,7 +115,6 @@ public class AmplitudeSession: EventPlugin, iOSLifecycle {
     }
 
     public func applicationWillResignActive(application: UIApplication?) {
-        stopTimer()
```

Comment:
May be worth putting a comment in here...
`// Exposed if reacting to lifecycle events is needed`

Reply:
Done